<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBasic_Info">
    <property name="title">
     <string>Basic Info</string>
    </property>
    <addaction name="actionVersion"/>
    <addaction name="actionBuildconf"/>
    <addaction name="separator"/>
    <addaction name="actionFormats"/>
    <addaction name="actionMuxers"/>
    <addaction name="actionDemuxers"/>
    <addaction name="actionDevices"/>
    <addaction name="separator"/>
    <addaction name="actionCodecs"/>
    <addaction name="actionDecoders"/>
    <addaction name="actionEncoders"/>
    <addaction name="separator"/>
    <addaction name="actionBsfs"/>
    <addaction name="actionProtocols"/>
    <addaction name="actionFilters"/>
    <addaction name="actionPixfmts"/>
    <addaction name="actionLayouts"/>
    <addaction name="actionSamplefmts"/>
    <addaction name="actionColors"/>
    <addaction name="separator"/>
    <addaction name="actionVideoRate"/>
    <addaction name="actionVideoSize"/>
    <addaction name="separator"/>
    <addaction name="actionSources"/>
    <addaction name="actionSinks"/>
    <addaction name="separator"/>
    <addaction name="actionL"/>
   </widget>
   <widget class="QMenu" name="menuMedia_Info">
    <property name="title">
     <string>Media Info</string>
    </property>
    <widget class="QMenu" name="menuFrames">
     <property name="title">
      <string>Frames</string>
     </property>
     <addaction name="actionShow_Frames_Video"/>
     <addaction name="actionShow_Frames_Audio"/>
    </widget>
    <widget class="QMenu" name="menuPackets">
     <property name="title">
      <string>Packets</string>
     </property>
     <addaction name="actionShow_Packets_Video"/>
     <addaction name="actionShow_Packets_Audio"/>
    </widget>
    <addaction name="actionShow_Streams"/>
    <addaction name="actionShow_Format"/>
    <addaction name="separator"/>
    <addaction name="menuFrames"/>
    <addaction name="menuPackets"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Entries"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Chapters"/>
    <addaction name="actionPrograms"/>
    <addaction name="separator"/>
    <addaction name="actionCount_Frames"/>
    <addaction name="actionCount_Packets"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Versions"/>
    <addaction name="actionShow_Program_Version"/>
    <addaction name="actionShow_Library_Versions"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Pixel_Formats"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPlay">
    <property name="title">
     <string>Play</string>
    </property>
    <addaction name="actionVLC"/>
    <addaction name="actionMPV"/>
    <addaction name="actionFFplay"/>
    <addaction name="separator"/>
    <addaction name="actionSMPlayer"/>
    <addaction name="actionTotem"/>
    <addaction name="actionGNOME_MPlayer"/>
    <addaction name="actionKMPlayer"/>
    <addaction name="actionXine"/>
    <addaction name="separator"/>
    <addaction name="actionInstall_Players"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Files"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDocs_Viewer"/>
    <addaction name="separator"/>
    <addaction name="actionApp_Dir"/>
    <addaction name="actionSetting_Dir"/>
    <addaction name="actionSetting_File"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionFFmpeg_Builder"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFiles"/>
    <addaction name="actionFolders"/>
    <addaction name="separator"/>
    <addaction name="actionLog"/>
    <addaction name="actionPlayer"/>
    <addaction name="separator"/>
    <addaction name="actionMedia_Properties"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBasic_Info"/>
   <addaction name="menuMedia_Info"/>
   <addaction name="menuPlay"/>
   <addaction name="menuView"/>
   <addaction name="menuConfig"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionBuildconf">
   <property name="text">
    <string>Buildconf</string>
   </property>
  </action>
  <action name="actionFormats">
   <property name="text">
    <string>Formats</string>
   </property>
  </action>
  <action name="actionMuxers">
   <property name="text">
    <string>Muxers</string>
   </property>
  </action>
  <action name="actionDemuxers">
   <property name="text">
    <string>Demuxers</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="text">
    <string>Devices</string>
   </property>
  </action>
  <action name="actionCodecs">
   <property name="text">
    <string>Codecs</string>
   </property>
  </action>
  <action name="actionDecoders">
   <property name="text">
    <string>Decoders</string>
   </property>
  </action>
  <action name="actionEncoders">
   <property name="text">
    <string>Encoders</string>
   </property>
  </action>
  <action name="actionBsfs">
   <property name="text">
    <string>Bsfs</string>
   </property>
  </action>
  <action name="actionProtocols">
   <property name="text">
    <string>Protocols</string>
   </property>
  </action>
  <action name="actionFilters">
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
  <action name="actionPixfmts">
   <property name="text">
    <string>Pixfmts</string>
   </property>
  </action>
  <action name="actionLayouts">
   <property name="text">
    <string>Layouts</string>
   </property>
  </action>
  <action name="actionSamplefmts">
   <property name="text">
    <string>SampleFmts</string>
   </property>
  </action>
  <action name="actionColors">
   <property name="text">
    <string>Colors</string>
   </property>
  </action>
  <action name="actionProgramVersion">
   <property name="text">
    <string>ProgramVersion</string>
   </property>
  </action>
  <action name="actionLibraryVersions">
   <property name="text">
    <string>LibraryVersions</string>
   </property>
  </action>
  <action name="actionFrame">
   <property name="text">
    <string>Video Frame</string>
   </property>
  </action>
  <action name="actionShow_Chapters">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Chapters</string>
   </property>
  </action>
  <action name="actionShow_Format">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Format</string>
   </property>
  </action>
  <action name="actionCount_Frames">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Count Frames</string>
   </property>
  </action>
  <action name="actionCount_Packets">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Count Packets</string>
   </property>
  </action>
  <action name="actionShow_Entries">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Entries</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionOpen_Files">
   <property name="text">
    <string>Open Files</string>
   </property>
  </action>
  <action name="actionShow_Streams">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Streams</string>
   </property>
  </action>
  <action name="actionShow_Program_Version">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Program Version</string>
   </property>
  </action>
  <action name="actionShow_Library_Versions">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Library Versions</string>
   </property>
  </action>
  <action name="actionShow_Versions">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Versions</string>
   </property>
  </action>
  <action name="actionShow_Pixel_Formats">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pixel Formats</string>
   </property>
  </action>
  <action name="actionLog_Conifg">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionMainwindow_Config">
   <property name="text">
    <string>Mainwindow</string>
   </property>
  </action>
  <action name="actionWindowHeader_Config">
   <property name="text">
    <string>Window Head</string>
   </property>
  </action>
  <action name="actionGeneral_Config">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionShow_Frames_Video">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionShow_Frames_Audio">
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionShow_Packets_Video">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionShow_Packets_Audio">
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPlayer">
   <property name="text">
    <string>Player</string>
   </property>
  </action>
  <action name="actionMedia_Properties">
   <property name="text">
    <string>Media Properties</string>
   </property>
  </action>
  <action name="actionFiles">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="actionFolders">
   <property name="text">
    <string>Folders</string>
   </property>
  </action>
  <action name="actionApp_Dir">
   <property name="text">
    <string>App Dir</string>
   </property>
  </action>
  <action name="actionSetting_Dir">
   <property name="text">
    <string>Setting Dir</string>
   </property>
  </action>
  <action name="actionSetting_File">
   <property name="text">
    <string>Setting File</string>
   </property>
  </action>
  <action name="actionL">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionFFmpeg_Builder">
   <property name="text">
    <string>Configure Builder</string>
   </property>
  </action>
  <action name="actionDocs_Viewer">
   <property name="text">
    <string>Docs Viewer</string>
   </property>
  </action>
  <action name="actionPrograms">
   <property name="text">
    <string>Programs</string>
   </property>
  </action>
  <action name="actionVideoRate">
   <property name="text">
    <string>VideoRate</string>
   </property>
  </action>
  <action name="actionVideoSize">
   <property name="text">
    <string>VideoSize</string>
   </property>
  </action>
  <action name="actionSources">
   <property name="text">
    <string>Sources</string>
   </property>
  </action>
  <action name="actionSinks">
   <property name="text">
    <string>Sinks</string>
   </property>
  </action>
  <action name="actionVLC">
   <property name="text">
    <string>VLC Media Player</string>
   </property>
  </action>
  <action name="actionMPV">
   <property name="text">
    <string>MPV Player</string>
   </property>
  </action>
  <action name="actionFFplay">
   <property name="text">
    <string>FFplay</string>
   </property>
  </action>
  <action name="actionSMPlayer">
   <property name="text">
    <string>SMPlayer</string>
   </property>
  </action>
  <action name="actionTotem">
   <property name="text">
    <string>Totem (GNOME Videos)</string>
   </property>
  </action>
  <action name="actionGNOME_MPlayer">
   <property name="text">
    <string>GNOME MPlayer</string>
   </property>
  </action>
  <action name="actionKMPlayer">
   <property name="text">
    <string>KMPlayer</string>
   </property>
  </action>
  <action name="actionXine">
   <property name="text">
    <string>Xine</string>
   </property>
  </action>
  <action name="actionInstall_Players">
   <property name="text">
    <string>Install Players...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
